pkgs/games/multimc/default.nix

```diff
@@ -14,7 +13,13 @@ in mkDerivation rec {
     fetchSubmodules = true;
   };
   nativeBuildInputs = [ cmake file makeWrapper ];
-  buildInputs = [ qtbase jdk zlib ];
+  buildInputs = [ qtbase jdk8 zlib ];
+
+  postPatch = ''
```

Comment:
Doesn't `JavaUtils.cpp` prioritize java located in `PATH` over everything else? https://github.com/MultiMC/MultiMC5/blob/develop/api/logic/java/JavaUtils.cpp#L353.

I think if you add `jdk16` to PATH like before, you shouldn't need this patch.

Reply:
If we add two `java` binaries to PATH, multimc(not only) will pick the first one. We should provide both versions of `java` as lots of people plays on older versions.

Reply:
`jre16` should be backwards compatible with `jre8`, so you should only need `jre16`. I'm assuming you're not using `jdk16` to build though since it fails to build with it?

Reply:
I will have to test this to ensure it doesn't break things like modded forge installations. There are plenty of older minecraft configurations (especially modded ones) that rely on the hotspot jre version 8 from oracle.